images/kcp-registrar/register.sh

```diff
@@ -132,12 +121,25 @@ register() {
     done
 }
 
+print_binding_manifest(){
```

Comment:
If the user uses PAC and triggers our kcp-registrar pipeline which would use this script, he won't be seeing any of the logs of the script unless he logs into the console to check them. Do you think its required to print this? I don't see any use as we have already documented in gitops/README on APIBinding and APIExport

Reply:
I thought it would be helpful to put the information there as well, but since that's not really related to this PR, I removed it.